src/configurationProvider.ts

```diff
@@ -94,7 +94,13 @@ export class JavaDebugConfigurationProvider implements vscode.DebugConfiguration
                     // for opened with folder, return directly.

```

Comment:
Totally don't understand the logic at `L92 ~ L96`, can you help update the comment?